packages/components/src/styles/@hashicorp/design-system-power-select-overrides.scss

```diff
@@ -167,6 +167,7 @@
 
   .ember-power-select-options {
     padding: 6px 0;
+    list-style: none;
```

Comment:
There was a stray bullet displaying.

Reply:
indeed, this is something I've seen happening since we upgraded to `ember-power-select` version 7 – we should cherry-pick this and open a PR to fix it. clearly a disadvantage of providing the styles without being in control of the version